recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# Architecture-dependent flags
+if [ "$(uname)" = "Darwin" ]; then
+    LIB_FLAG='-dynamiclib'
+    EXT='.dylib'
+    deployFlag="-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13"
+else
+    LIB_FLAG='-shared'
+    EXT='.so'
+    deployFlag=""
+fi
+
+CMAKE_ARGS+=("-DCMAKE_PREFIX_PATH=${PREFIX}")
+
+# Build dependencies first
+
+# Multiview
+cd $SRC_DIR/MultiView
+mkdir -p build && cd build
+cmake ..                                          \
+    -DCMAKE_BUILD_TYPE=Release                    \
+    -DMULTIVIEW_DEPS_DIR=${PREFIX}                \
+    -DCMAKE_VERBOSE_MAKEFILE=ON                   \
+    -DCMAKE_CXX_FLAGS="-O3 -std=c++11 -Wno-error" \
+    -DCMAKE_C_FLAGS='-O3 -Wno-error'              \
```

Comment:
Same as above. The default here is `${CFLAGS}`.

Reply:
I removed -DCMAKE_C_FLAGS. I need to keep the C++ flags until I find time to review and overhaul the internals of these.